application/common/config/main.php

```diff
@@ -2,8 +2,13 @@
 
 use Sil\Idp\IdSync\common\components\IdBrokerBase;
 use Sil\Idp\IdSync\common\components\IdStoreBase;
+use Sil\JsonSyslog\JsonSyslogTarget;
 use Sil\PhpEnv\Env;
 use yii\helpers\ArrayHelper;
+use yii\helpers\Json;
+
+$APP_ENV = Env::get('APP_ENV', 'prod'); // Have default match "application/frontend/web/index.php".
+$IDP_NAME = Env::get('IDP_NAME');
```

Comment:
Which metadata? I think this is just used (in this case) for providing richer log data, in which case just showing `null` when it's not present seemed preferable.